server/middleware/validations/skill-validation.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+let ErrorFactory = require('../../resources').ErrorFactory;
+let Errors = require('../../resources').Errors;
+let topicFields = require('../../resources').fields;
+
+const validateSkill = (req, res, next) => {
+    let skill = req.body;
+
+    let requiredFields = [
+        'name',
+        'experience',
+        'field',
+    ];
+    
+    // Should have skill
+    if(Object.keys(skill).length === 0) {
+        let error = ErrorFactory.buildError(Errors.MISSING_FIELD, 'skill (object)');
+        return res.status(error.status).send({ error: error });
+    }
+
+    // Required fields
+    for(let field of requiredFields) {

```

Comment:
Why do we need to provide `field` with skill? isn't it part of topic, which we are referencing by id?